### PR TITLE
Add images

### PR DESCRIPTION
**Pull request by Grub2801**

### CODE REVIEW

File: app/models/review.rb [hunk 1]
```diff
@@ -2,6 +2,10 @@ class Review < ActiveRecord::Base
   belongs_to :user
   belongs_to :playspace
 
+  has_attached_file :photo, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
```
Comment (f3r) [inline app/models/review.rb]: you should add a `missing.png` photo for each style: `original`, `medium`, `thumb`

File: app/views/playspaces/index.html.erb [hunk 2]
```diff
@@ -10,6 +10,12 @@
           <% else %>
             <% @playspaces.each do |playspace| %>
               <li class="col-xs-12 col-sm-4 col-md-3 add-padding">
+                <div>
+                  <%= image_tag(playspace.photo_file) %>
```
Comment (f3r) [inline app/views/playspaces/index.html.erb]: I believe it's
```erb
<%= image_tag(playspace.photo_url) %>
```

File: db/migrate/20151105025101_add_photo_column_to_playspaces.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddPhotoColumnToPlayspaces < ActiveRecord::Migration
+  def change
+    add_column :playspaces, :photo_file, :string
```
Comment (f3r) [inline db/migrate/20151105025101_add_photo_column_to_playspaces.rb]:
```
add_attachment :playspaces, :photo
```

File: app/assets/stylesheets/playspaces.scss [hunk 4]
```diff
@@ -15,6 +15,17 @@
...
   }
+
+  div > img {
+    height: auto;
+    width: 100%;
+    border-radius: 5px;
+    background-position: center;
+  }
+
+  .floated_img {
```
Comment (f3r) [inline app/assets/stylesheets/playspaces.scss]: you should use `pull-left`

File: app/assets/stylesheets/playspaces.scss [hunk 5]
```diff
@@ -15,6 +15,17 @@
...
   }
+
+  div > img {
+    height: auto;
+    width: 100%;
+    border-radius: 5px;
+    background-position: center;
+  }
+
+  .pull-left {
```
Comment (f3r) [inline app/assets/stylesheets/playspaces.scss]: @Grub2801 the class `pull-left` is in bootstrap already :)

File: db/seeds.rb [hunk 6]
```diff
@@ -65,7 +65,8 @@
   carpark: true,
   snack_shop: true,
   wc: true,
-  category: Category.find_by(name: "park")
+  category: Category.find_by(name: "park"),
+  photo_file: "hkpark1.jpg"
```
Comment (f3r) [inline db/seeds.rb]: to add an image from the seeds, you should have something like:
```rb
Playspace.create(name: "hong kong park",
  ..
  photo: File.open("#{Rails.root.join('app', 'assets', 'images', 'hkpark1.jpg')}")
```

**Comment by f3r** [comment]
Please review changes and let me know when to review

**Comment by Grub2801** [comment]
@f3r changes made

**Pull request merged**